src/components/AccountAssets/AccountAssets.tsx

```diff
@@ -22,7 +22,7 @@ export const AccountAssets = ({ assetId, accountId }: AccountAssetsProps) => {
   const asset = useAppSelector(state => selectAssetById(state, assetId))
 
   // @TODO: This filters for ETH to not show tokens component on tokens
-  if (asset.tokenId || assetIds.length === 0) return null
+  if (!(asset.symbol === 'ETH') || assetIds.length === 0) return null
```

Comment:
use asset reference rather than eth?

Reply:
Addressed in https://github.com/shapeshift/web/pull/1908/commits/7cd8a1b29941e6cf011fe4054a4052058314b857